<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageClientsView</class>
 <widget class="QMainWindow" name="ManageClientsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(232, 210, 184);
font: 10pt &quot;URW Gothic L&quot;;</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="leftLayout" stretch="0,0,0,0">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="clientTitleLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Noto Sans Devanagari UI</family>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 15pt &quot;Noto Sans Devanagari UI&quot;;
font: bold</string>
        </property>
        <property name="text">
         <string>Clients</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="clientTableView">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>221</width>
          <height>330</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>221</width>
          <height>330</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="tempLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="maximumSize">
         <size>
          <width>202</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="rightLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="titleDetailsLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Noto Sans Devanagari UI</family>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 15pt &quot;Noto Sans Devanagari UI&quot;;
font: bold</string>
        </property>
        <property name="text">
         <string>Details</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="basicTab">
         <attribute name="title">
          <string>Basic</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>391</width>
            <height>301</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="clientDetailsLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::FieldsStayAtSizeHint</enum>
           </property>
           <property name="labelAlignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="formAlignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="horizontalSpacing">
            <number>5</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <item row="1" column="0">
            <widget class="QLabel" name="idLabel">
             <property name="text">
              <string>ID : </string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="displayID">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="nameLabel">
             <property name="text">
              <string>Name : </string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="displayName">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="emailLabel">
             <property name="text">
              <string>Email : </string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="displayEmail">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="addressLabel">
             <property name="text">
              <string>Address : </string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="displayAddress">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="phoneLabel">
             <property name="text">
              <string>Phone no. : </string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLabel" name="displayPhoneNo">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="personalityTab">
         <attribute name="title">
          <string>Personality</string>
         </attribute>
         <widget class="QWidget" name="formLayoutWidget">
          <property name="geometry">
           <rect>
            <x>-1</x>
            <y>-1</y>
            <width>391</width>
            <height>321</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="extraversionLabel">
             <property name="text">
              <string>Extraversion :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="displayExtraversion">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="agreeablenessLabel">
             <property name="text">
              <string>Agreeableness :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="displayAgreeableness">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="conscientiousnessLabel">
             <property name="text">
              <string>Conscientiousness :</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="neuroticismLabel">
             <property name="text">
              <string>Neuroticism :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="displayConscientiousness">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="displayNeuroticism">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="opennesLabel">
             <property name="text">
              <string>Openness :</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="displayOpenness">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Matching Preference</string>
         </attribute>
         <widget class="QWidget" name="formLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>-1</x>
            <y>-1</y>
            <width>401</width>
            <height>321</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="ageLabel">
             <property name="text">
              <string>Age :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="displayAge">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="genderLabel">
             <property name="text">
              <string>Gender :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="displayGender">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="typeLabel_2">
             <property name="text">
              <string>Type :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="displayType_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="sizeLabel">
             <property name="text">
              <string>Size :</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="displaySize">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="kidFriendLabel">
             <property name="text">
              <string>Kid-Friendliness :</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="displayKidFriendliness">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="costlabel">
             <property name="text">
              <string>Cost :</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLabel" name="displayCost">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="currenTrainingLabel">
             <property name="text">
              <string>Current Training :</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLabel" name="displayCurrentTraining">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="intelligenceLabel">
             <property name="text">
              <string>Intelligence :</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLabel" name="displayIntelligence">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="timeCommLabel">
             <property name="text">
              <string>Time Commitment :</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QLabel" name="displayTimeComm">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="sheddingLabel">
             <property name="text">
              <string>Shedding :</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="apartmentLivingLabel">
             <property name="text">
              <string>Apartment-Living :</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QLabel" name="displayShedding">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="QLabel" name="displayApartmentCompatibility">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QLabel" name="trainingSuitabilityLabel">
             <property name="text">
              <string>Training Suitability :</string>
             </property>
            </widget>
           </item>
           <item row="12" column="1">
            <widget class="QLabel" name="displayTrainingSuitability">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
